podium-uaa/src/main/resources/config/application.yml

```diff
@@ -85,3 +85,9 @@ podium:
     ribbon:
         # comma separated list of profiles to show a ribbon on the GUI
         displayOnActiveProfiles: dev
+
+uaa:
+    security:
+        maxFailedLoginAttempts:
```

Comment:
@berndvdveen This does not work. Any idea how an empty entry could be added?
This results in `2017-02-23 16:12:09.981 ERROR 5861 --- [           main] o.s.b.b.PropertiesConfigurationFactory   : Field error in object 'uaa' on field 'security.maxFailedLoginAttempts': rejected value []; codes [typeMismatch.uaa.security.maxFailedLoginAttempts,typeMismatch.security.maxFailedLoginAttempts,typeMismatch.maxFailedLoginAttempts,typeMismatch.long,typeMismatch]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [uaa.security.maxFailedLoginAttempts,security.maxFailedLoginAttempts]; arguments []; default message [security.maxFailedLoginAttempts]]; default message [Failed to convert property value of type 'java.lang.String' to required type 'long' for property 'security.maxFailedLoginAttempts'; nested exception is org.springframework.core.convert.ConverterNotFoundException: No converter found capable of converting from type [java.lang.String] to type [long]]`